### PR TITLE
Add litchi to fruits

### PR DESCRIPTION
**Pull request by anuseth12**

### CODE REVIEW

File: src/listInterface/ArrayListExampke.java [hunk 1]
```diff
@@ -18,9 +18,10 @@ public static void main(String[] args) {
   
   
 	        // Adding elements
-	        fruits.add("Apple");
+	        fruits.add("Lichi");
```
Comment (Jagadeesh-babu-cloud) [inline src/listInterface/ArrayListExampke.java]: As discussed please remove this and add watermelon

File: src/listInterface/Listinsidelist.java [hunk 2]
```diff
@@ -43,7 +43,8 @@ public static List<List<Integer>> generate(int numRows) {
            sub1.addAll(sub);
 	       l.add(new ArrayList<>(sub));
 	       
-	           System.out.println("sub list is"+ sub);	          System.out.println("main list is"+ l);
+	           System.out.println("sub list is"+ sub);
```
Comment (Jagadeesh-babu-cloud) [inline src/listInterface/Listinsidelist.java]: not needed, pls remove

File: src/listInterface/ArrayListExampke.java (hunk 1 above)
Comment (anuseth12) [reply src/listInterface/ArrayListExampke.java]: ok added

File: src/listInterface/Listinsidelist.java (hunk 2 above)
Comment (anuseth12) [reply src/listInterface/Listinsidelist.java]: removed

**Review by Jagadeesh-babu-cloud — APPROVED** [review]
looks good, please merge

**Pull request merged**